src/many-ledger/src/module/ledger_mintburn.rs

```diff
@@ -102,3 +92,26 @@ impl ledger::LedgerMintBurnModuleBackend for LedgerModuleImpl {
             .map(|_| TokenBurnReturns { distribution })
     }
 }
+
+impl LedgerModuleImpl {
+    /// Only the token identity, the server identity or the token owner is allowed to mint/burn
+    fn verify_mint_burn_identity(
+        &mut self,
+        sender: &Address,
+        symbol: &Symbol,
+    ) -> Result<(), ManyError> {
+        // Are we the token identity or the server identity?
```

Comment:
I dont think the server identity makes sense here

Reply:
The token identity is currently optional in the ledger staging file. The default is to use the server identity to stay backward compatible.

We should refactor this in a future PR.

See #375